--- a/rootfs/etc/crontab
+++ b/rootfs/etc/crontab
@@ -12,4 +12,5 @@ PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
 25 6   * * *   root    test -x /usr/sbin/anacron || ( cd / && run-parts --report /etc/cron.daily )
 47 6   * * 7   root    test -x /usr/sbin/anacron || ( cd / && run-parts --report /etc/cron.weekly )
 52 6   1 * *   root    test -x /usr/sbin/anacron || ( cd / && run-parts --report /etc/cron.monthly )
-#
+* *     * * *   root    /usr/local/bin/client-services --sync_state_to_portal
+* *     * * *   root    /usr/local/bin/client-services --update_stats_file